table.scss

```diff
@@ -0,0 +1,126 @@
+	}
+
+	& tr {
+		display: table-row;
+	}
+
+	& td {
+		@include vui-table-cell();
+	}
+
+	& tr td:last-child {
+		border-right: none;
+	}
+
+	& tr:last-child td {
+		border-bottom: none;
+	}
+
+	& tbody th,
+	  tfoot th {
+		@include vui-table-cell();
+	}
+
+	& tr th:last-child {
+		border-right: none;
+	}
+
+	//Correct for corners bleeding outside the rounded corners
+	& thead > tr:first-child > :first-child,
+	  tbody:only-child > tr:first-child > :first-child,
+	  tfoot:first-child + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + tfoot + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + colgroup + tbody:last-child > tr:first-child > :first-child,
+	  caption:first-child + colgroup + tfoot + tbody:last-child > tr:first-child > :first-child,
+	  colgroup:first-child + tbody:last-child > tr:first-child > :first-child,
+	  colgroup:first-child + tfoot + tbody:last-child > tr:first-child > :first-child {
+		border-top-left-radius: 6px;
+	}
+	& thead > tr:first-child > :last-child,
+	  tbody:only-child > tr:first-child > :last-child,
+	  tfoot:first-child + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + tfoot + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + colgroup + tbody:last-child > tr:first-child > :last-child,
+	  caption:first-child + colgroup + tfoot + tbody:last-child > tr:first-child > :last-child,
+	  colgroup:first-child + tbody:last-child > tr:first-child > :last-child,
+	  colgroup:first-child + tfoot + tbody:last-child > tr:first-child > :last-child {
+		border-top-right-radius: 6px;
+	}
+	& tfoot > tr:last-child > :first-child,
+	  tbody:only-child > tr:last-child > :first-child,
+	  thead + tbody:last-child > tr:last-child > :first-child,
+	  caption + tbody:last-child > tr:last-child > :first-child,
+	  colgroup + tbody:last-child > tr:last-child > :first-child {
+		border-bottom-left-radius: 6px;
+	}
+	& tfoot > tr:last-child > :last-child,
+	  tbody:only-child > tr:last-child > :last-child,
+	  thead + tbody:last-child > tr:last-child > :last-child,
+	  caption + tbody:last-child > tr:last-child > :last-child,
+	  colgroup + tbody:last-child > tr:last-child > :last-child {
+		border-bottom-right-radius: 6px;
+	}
+
+	& tfoot tr:last-child th,
+	  tbody:last-child>tr:last-child>th {
+			border-bottom: none;
+	}
+
+	//Apply hover to only rows that are not contained in thead, tfoot or have the class .vui-table-header or .vui-table-footer
+	& > tbody > tr:hover:not(.vui-table-header):not(.vui-table-footer) {
+		outline: 1px solid $vui-color-celestine;
+		background-color: $vui-color-celestine-light-1;
+	}
+}
+
+@mixin vui-table-header-cell() {
+	@include vui-typography-small-text();
+	@include vui-table-cell();
+	background-color: $vui-color-regolith;
+	font-weight: 400;
```

Comment:
[vui-typography-small-text](https://github.com/Brightspace/valence-ui-typography/blob/master/typography.scss#L97) specifies font-weight 400 already, so I don't think you'll need to set this.  It also accepts and argument if you need to override font-weight.